examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    detuning_value = amplitude_value  # rad / s
</pre><p>What is the purpose of non-zero detuning? (Sorry, I probably don't understand the intention behind this AHS program.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>